src/mocks/contributorRole.ts

```diff
@@ -1,37 +1,105 @@
-// Just define what is unique here. Any fields you skip will end up using the defaults
-// for their respective type as defined in ./src/mocks.ts
-const data = [
+import { Resolvers } from "../types";
+import { MockMySQLTable } from './MockMySQLTable';
+// import { ContributorRole, ContributorRoleMutationResponse } from '../types';
+
+// Seed records for the ContributorRoles table
+const mockItems = [
   {
-    id: 1,
+    displayOrder: 1,
```

Comment:
L8: The other objects in the array  have "displayOrder" property last. Should we keep them consistent?